### DISCUSSION TITLE
How to deal with illegal actions/terrorism?

### DISCUSSION THREAD

**Discussion by mrbaranovskiy** [community/community#38283] [Other Feature Feedback, Questions, & Ideas] [answered]
I found out that russian 'hackers' created a public repository to hack Ukraine's cyber security space/software? In the same time, calling GitHub enemies space. 

I'm not sure this follows the community rules.

Link to repo. https://github.com/RusItFront

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mrbaranovskiy 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
